### DISCUSSION TITLE
How To Install

### DISCUSSION THREAD

**Discussion by OrdinaryPerson1** [skyline-emu/skyline#813] [General]
I might be missing something obvious here, but I can't install this. I'm looking into the files and haven't seen an APK file. Would I need to compile it INTO an APK? If so, could anyone please give me info on how? Thank you!

**Comment by ghost** [comment]
Go to actions tab, click latest from ftx1 branch, make sure u are logged in and grab app-release.apk

**Reply by OrdinaryPerson1** [reply]
Thank you, I didn't expect this quick of an answer!
